code/API_definitions/kyc-fill-in.yaml

```
@@ -84,6 +84,17 @@ paths:
       parameters:
         - $ref: '#/components/parameters/x-correlator'
 
+      requestBody:
+        content:
+          application/json:
+
+            schema:
+              $ref: '#/components/schemas/KYC_FillinRequest'
+            examples:
+              KYC_FillInBodyExample:
+                value:
+                  phoneNumber: '+34629255833'

```

The content-type is application/json but the example is not JSON.
```suggestion
                  { "phoneNumber": "+34629255833" }

```